CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
-## [5.5.0](https://github.com/terraform-aws-modules/terraform-aws-redshift/compare/v5.4.0...v5.5.0) (2024-05-08)
+## [6.0.0](https://github.com/terraform-aws-modules/terraform-aws-redshift/compare/v5.4.0...v6.0.0) (2024-05-08)
```

Comment:
Isn't changelog regenerating every time by during release from actual git data?

Reply:
hmm, I don't know. should I let this be and see what happens after the next release?

Reply:
Yes, it will be corrected on next release if you have removed incorrect git tag.